lib/plans/checkout.dart

```diff
@@ -69,8 +69,6 @@ class _CheckoutState extends State<Checkout>
 
   @override
   void dispose() {
-    emailController.dispose();
-    refCodeController.dispose();
```

Comment:
We are already doing dispose on CTextField, Doing it twice was throwing error.

Reply:
Thanks for fixing this!